catalyst-gateway/bin/src/service/api/cardano/staking/assets_get.rs

```diff
@@ -100,22 +97,21 @@ pub(crate) async fn endpoint(
     .into()
 }
 
-/// TXO asset information.
-struct TxoAssetInfo {
+/// TXO Native Token information.
+struct NativeTokens {
     /// Asset hash.
     id: Vec<u8>,
     /// Asset name.
     name: AssetName,
     /// Asset amount.
+    #[allow(dead_code)]
     amount: num_bigint::BigInt,
```

Comment:
is this amount redundant now?

Reply:
dead_code redundant

Reply:
How is it redundant?

Reply:
no no, I meant `#[allow(dead_code)]` line is redundant